src/Compilers/VisualBasic/Portable/Syntax/SyntaxFactory.vb

```diff
@@ -748,7 +748,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 XmlAttribute(
                     XmlName(
                         Nothing,
-                        XmlTextLiteralToken(DocumentationCommentXmlNames.KeywordElementName, DocumentationCommentXmlNames.KeywordElementName)),
+                        XmlTextLiteralToken(DocumentationCommentXmlNames.LangwordAttributeName, DocumentationCommentXmlNames.LangwordAttributeName)),
```

Comment:
:memo: You didn't introduce the issue, but this should be an XmlNameToken token:

```
Token(SyntaxKind.XmlNameToken, DocumentationCommentXmlNames.LangwordAttributeName)
```